src/purchases.ts

```diff
@@ -185,24 +195,25 @@ export default class Purchases {
    * Sets up Purchases with your API key and an app user id.
    * @param {String} apiKey RevenueCat API Key. Needs to be a String
    * @param {String?} appUserID An optional unique id for identifying the user. Needs to be a string.
-   * @param {boolean} [observerMode=false] An optional boolean. Set this to TRUE if you have your own IAP implementation and want to use only RevenueCat's backend. Default is FALSE.
-   * @param {boolean} [usesStoreKit2IfAvailable=false] DEPRECATED. An optional boolean. iOS-only. Defaults to FALSE. Setting this to TRUE will enable StoreKit2 on compatible devices.
-   * We recommend not using this parameter, letting RevenueCat decide for you which StoreKit implementation to use.
+   * @param {PURCHASES_ARE_COMPLETED_BY} [purchasesAreCompletedBy=PURCHASES_ARE_COMPLETED_BY.REVENUECAT] Set this to PURCHASES_ARE_COMPLETED_BY.MY_APP if you have your own IAP implementation and want to use only RevenueCat's backend. Default is PURCHASES_ARE_COMPLETED_BY.REVENUECAT.
+   * @param {STOREKIT_VERSION} [storeKitVersion=DEFAULT] iOS-only. Defaults to STOREKIT_2. StoreKit 2 is only available on iOS 16+. StoreKit 1 will be used for previous iOS versions regardless of this setting.
    * @param {ENTITLEMENT_VERIFICATION_MODE} [entitlementVerificationMode=ENTITLEMENT_VERIFICATION_MODE.DISABLED] Sets the entitlement verifciation mode to use. For more details, check https://rev.cat/trusted-entitlements
    * @param {boolean} [useAmazon=false] An optional boolean. Android-only. Set this to TRUE to enable Amazon on compatible devices.
    * @param {String?} userDefaultsSuiteName An optional string. iOS-only, will be ignored for Android.
    * Set this if you would like the RevenueCat SDK to store its preferences in a different NSUserDefaults suite, otherwise it will use standardUserDefaults.
    * Default is null, which will make the SDK use standardUserDefaults.
+   * @param {boolean} [pendingTransactionsForPrepaidPlansEnabled=false] An optional boolean. Android-only. Set this to TRUE to enable pending transactions for prepaid subscriptions in Google Play.
    */
   public static configure({
     apiKey,
     appUserID = null,
-    observerMode = false,
+    purchasesAreCompletedBy = PURCHASES_ARE_COMPLETED_BY.REVENUECAT,
     userDefaultsSuiteName,
-    usesStoreKit2IfAvailable = false,
```

Comment:
Not sure if we also want to deprecate this one or forcibly remove it TBH... IMO, we should try to deprecate it instead of removing unless there is a reason for removal

Reply:
I see we actually removed this one from the TS interface for `PurchasesConfiguration` in PHC, so adding it deprecated would require another PHC release... Maybe it's ok to just remove it?

Reply:
Yeah, we've completely removed the `usesStoreKit2IfAvailable` param from the natives and want everyone to be using the STOREKIT_VERSION enum going forward, so it's okay to remove it here!